### PR TITLE
Update README for new CLI

### PR DESCRIPTION
**Pull request by qihqi**
Update README to refer to new CLI instead of old scripts. 

This README is not complete yet. But it will be completed with more details on flags before next release.

### CODE REVIEW

**Review by lsy323 — APPROVED** [review]

File: jetstream_pt/fetch_models.py [hunk 1]
```diff
@@ -38,15 +38,18 @@ class ModelInfo:
   model_class: torch.nn.Module
   # information needed to allocate cache
   num_layers: int
+  # number of kv heads
   num_heads: int
```
Comment (wang2yn84) [inline jetstream_pt/fetch_models.py:42]: This is actually number of kv heads.

**Review by wang2yn84 — CHANGES_REQUESTED** [review]
In general good, some nits and small issues. Thank you!

**Review by FanhaiLu1 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -49,108 +45,69 @@ cd jetstream-pytorch
...
-* `--shard_on_batch=1` This makes the model to shard on 
-  the batch dimension. I.e. this runs in data parallel mode instead of model
-  parallel. This will ignore the sharding config. This is recommended for Gemma 2B
-  model, because Gemma 2B is small enough to fit on a single TPU chip.
+To run this model in `int8` quantization, add `--quantize_weights=1`.
+Quantization will be done on the flight as the weight loads.
 
-* `--sharding_config=<path>` This makes use of alternative sharding config instead of
-  the ones in default_shardings directory.
+Weights downloaded from HuggingFace will be stored by default in `checkpoints` folder.
```
[the remark this answers is not in the document]
Comment (qihqi) [reply README.md:109]: For gs bucket it need to be brought locally or use mount using Fuse.

The working dir can be edited. Added paragraph to describe that.

File: jetstream_pt/fetch_models.py (hunk 1 above)
Comment (qihqi) [reply jetstream_pt/fetch_models.py:42]: renamed var

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (wang2yn84) [reply README.md:109]: It'll be great if you can add how to change the working dir. Cuz for us, we also need to direct to the external ssd. I will approve the PR to unblock you for now.

**Review by wang2yn84 — APPROVED** [review]

**Pull request merged**